docs/analytics/jenkins_jobs.md

```diff
@@ -117,6 +117,20 @@ analytics-configuration/automation/run-automated-task.sh AnswerDistributionWorkf
 
 In edx docs: [Enrollment](http://edx-analytics-pipeline-reference.readthedocs.io/en/latest/running_tasks.html#enrollment)
 
+Before running the daily incremental enrollment task, below, you should first run the history task that loads historical enrollment events.
+
+```bash
+. /home/jenkins/jenkins_env
+export CLUSTER_NAME="ImportEnrollmentsIntoMysql_HistoryTask Cluster"
+cd $HOME
+
+TO_DATE=`date +%Y-%m-%d`
+analytics-configuration/automation/run-automated-task.sh CourseEnrollmentEventsTask \
+    --local-scheduler \
+    --interval "2013-01-01-$TO_DATE" \
```

Comment:
nit: I would instead have a `FROM_DATE=2013-01-01` parameter above, with a comment to say this date should be adjusted to the earliest date found in the client tracking logs.  After all, if they only started running in 2016, then we don't need to work through all those days.